sycl/test/basic_tests/opencl_interop.cpp

```diff
@@ -0,0 +1,83 @@
+// REQUIRES: opencl
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out -L %opencl_libs_dir -lOpenCL
```

Comment:
It seems the test is only compiled. Please, add run line which executes the test.